tracker/tracker/src/main/app/observer/observer.ts

```diff
@@ -225,9 +227,10 @@ export default abstract class Observer {
         false,
       )
       while (walker.nextNode()) {
+        removed += 1
         this.app.nodes.unregisterNode(walker.currentNode)
       }
-      // MBTODO: count and send RemovedNodesCount (for the page crash detection in heuristics)
+      this.app.send(RemovedNodesCount(removed))
```

Comment:
maybe add rootID + something to detect visual significance. Dimensions of root, for example

Reply:
we only want to use it in heuristics level to assume that there is UI crash when this event occurs near error event; not sure if we need node id here

Reply:
We might need rootID/selector etc. in the future, for example, for chasing the root of the problem. And updating messages would be costly at the moment.
And the "visual significance" might be an analysis criteria already. I am not sure though if it is easy to figure out the dimensions of a node that have been removed already - `getComputedStyle()` won't work at that moment, I suppose.

I am just discussing.